app/assets/stylesheets/views/_covid.scss

```diff
@@ -66,6 +66,10 @@ $c19-landing-page-header-background: #0843a1;
   @include govuk-media-query($from: tablet) {
     padding: 45px 0 40px;
   }
+
+  .gem-c-govspeak {
+    color: govuk-colour("white");
+  }
```

Comment:
Eep - is there a container div we could apply this to? it seems a bit of a no-no to override components directly.

Reply:
We seem to be doing it [rather](https://github.com/alphagov/collections/pull/2206/files/2323664398cdaf7715b61ae7e29d25d28a136fc4#diff-b599aa1768841fa36cef988b2e58f4bcc5fb93fe4681f31f5f43d8bd81510aa2R47-L49) [often](https://github.com/alphagov/collections/pull/2206/files/2323664398cdaf7715b61ae7e29d25d28a136fc4#diff-b599aa1768841fa36cef988b2e58f4bcc5fb93fe4681f31f5f43d8bd81510aa2L209-L211) – although I've heard voices saying we don't.

Reply:
Adding extra containers to apply the colour on won't work as `gem-c-govspeak` defines its own colour and will override it.

Reply:
Ah bummer - hmm well I guess there isn't time for a better approach now. I wonder what a better approach would be though.

Reply:
I raised the overriding topic in the #govukfrontenders a few times now – people are not interested – but happy to raise it again in the hope to reach a consensus. You probably know my stance by now: modifier classes should be allowed in components 🙂